db/providers/notes.js

```diff
@@ -0,0 +1,27 @@
+//db calls
+import notesSchema from '../../modules/notes/models/notesSchema.js'
+import { postfixName } from '../../env.js'
+import dotenv from 'dotenv'
+dotenv.config()
+export const noteGet = async () => {
+  const noteFind = await notesSchema.find().limit().sort({ createdAt: 1 })
+  return noteFind
+}
+
+export const noteCreate = async (newItem) => {
+  newItem.title = postfixName(newItem.title)
+  const noteSave = await notesSchema.create(newItem)
+  return noteSave
+}
+
+export const noteUpdate = async (_id, noteNew) => {
+  const noteUpd = await notesSchema.findByIdAndUpdate({ _id: _id }, noteNew, {
```

Comment:
Wht f i ll wrt the sm commnts?

Reply:
Replace it with something more clear!!!
```suggestion
  const updatedNote = await notesSchema.findByIdAndUpdate({ _id: _id }, details, {
```